### DISCUSSION TITLE
Strangely printed numbers of steps in the training log and mini-batching

### DISCUSSION THREAD

**Discussion by tobiassugandi** [lululxvi/deepxde#1489] [Q&A]
Hi everyone, 

I have a question regarding several strangely printed numbers of steps during training (see below). I didn't specify 'display_every' to the training function, so it should have been every 1000 steps. I am using DeepONet with Triplet data and I use mini-batch by specifying the 'batch_size' when the model is calling train(), i.e., `losshistory, train_state = model.train(batch_size = 2**16)`. I am wondering if anyone has any idea regarding this behavior.

> Step      Train loss    Test loss     Test metric
> 0         [1.23e+00]    [1.59e+00]    []  
> 1000      [1.00e-01]    [1.70e-01]    [] 
> ...
> 47000     [9.79e-05]    [2.59e-04]    []  
> 48000     [9.55e-05]    [2.55e-04]    []  
> 48675     [1.02e-04]    [2.60e-04]    []  
> 49675     [9.72e-05]    [2.44e-04]    []  
> ...
> 64675     [8.79e-05]    [2.48e-04]    []  
> 65489     [8.41e-05]    [2.41e-04]    [] 

I also wanted to clarify if I understood this correctly: Does the printed value of 'Step' mean the number of batch that has been used for training, e.g., Step 1000 means in my case '1000 * 2**16' training data has been used for training? 

Thanks a lot in advance!

**Comment by lululxvi** [comment]
Really strange. It should be 1000.
